ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -1156,3 +1164,28 @@ int OwInterface::actionGoalStatus (const string& action_name) const
 {
   return ActionGoalStatusMap[action_name];
 }
+
+double OwInterface::jointTelemetry (int joint, TelemetryType type) const
+{
+  if (joint >= 0 && joint < KnownJoints) {
+    switch (type) {
+      case TelemetryType::Position: return JointTelemetries[joint].position;
+      case TelemetryType::Velocity: return JointTelemetries[joint].velocity;
+      case TelemetryType::Effort: return JointTelemetries[joint].effort;
+      case TelemetryType::Acceleration: {
+        if (joint < ArmJointStartIndex) {
+          ROS_WARN ("jointTelemetry: acceleration not available for antenna joint.");
+          break;
+        }
+        else return JointTelemetries[joint].acceleration;
+      }
+      default:
+        ROS_ERROR ("jointTelemetry: unsupported telemetry type.");
+        break;
+    }
+  }
+  else {
+    ROS_ERROR ("jointTelemetry: invalid joint index %d", joint);
+  }
+  return 0;
```

Comment:
It doesn't, but good suggestion.  It supports UNKNOWN for every type -- I'll see if that could work here.

Reply:
Ah, I forgot about that feature. I think returning Unknown if the value cannot be looked up would make the most sense. Then users can verify their lookups with `isKnown`.

Reply:
Agreed, it would take a bit of work to support returning Unknown back up to PLEXIL, and I should apply it to all other similar situations.  Creating a new Jira for this.